ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/OVERLAY_EBGP_EVPN_GATEWAY_TESTS.yml

```
@@ -1,15 +1,93 @@
 type: l3leaf
 
+evpn_overlay_bgp_rtc: true
+
 l3leaf:
   defaults:
-    loopback_ipv4_pool: 192.168.255.0/24
-    vtep_loopback_ipv4_pool: 192.168.254.0/24
-    virtual_router_mac_address: 00:dc:00:00:00:0a
-    platform: vEOS
+    loopback_ipv4_pool: 169.254.102.0/24
+    vtep_loopback_ipv4_pool: 169.253.103.0/24
```

Hmmm 169.253 is not a good range for the IPs in our example - can you keep 169.254.103?
cf
https://en.wikipedia.org/wiki/Reserved_IP_addresses

```suggestion
    vtep_loopback_ipv4_pool: 169.253.103.0/24
```

also did you need to change?

---

Reverted to original